cmd/service.go

```diff
@@ -217,14 +220,20 @@ func runServiceStartCmd(cmd *cobra.Command, args []string) {
 	if viper.GetBool(constants.ArgDashboard) {
 		dashboardState, err = dashboardserver.GetDashboardServiceState()
 		if err != nil {
-			db_local.StopServices(ctx, false, constants.InvokerService)
+			_, err1 := db_local.StopServices(ctx, false, constants.InvokerService)
```

Comment:
wrap this in function and use for all similar cases

```
func tryToStopServices(ctx context.Context){
  if _, err := db_local.StopServices(ctx, false, constants.InvokerService); err != nil {
	utils.ShowError(ctx, err)
  }
}
```

Reply:
updated